src/pages/Check/index.jsx

```diff
@@ -13,12 +13,29 @@ const Check = () => {
   const history = useHistory()
   const { packages, loading, error } = useGetPackagesIdByCode(packageCode?.packageCode)
 
-  const toDelivery = async e => {
-    if (e?.packageCode && !error && !loading) {
-      const foundPackage = (await packages.length) > 0
+  // eslint-disable-next-line complexity
+  const toDelivery = async (/** @type {{ packageCode: String; }} */ event) => {
+    if (event?.packageCode && !error && !loading) {
+      const foundPackage = packages.length > 0
+
       if (foundPackage) {
-        const { id } = await packages[0]
-        history.push(`/delivery/${id}`)
+        const { id, order_status } = packages[0]
+        // eslint-disable-next-line camelcase
+        switch (order_status) {
```

Comment:
I think you can save order_status as orderStatus and delete the eslint comment

```suggestion
        const { id, order_status: orderStatus } = packages[0]

        switch (orderStatus) {
```